app/schema.graphql

```diff
@@ -491,14 +491,20 @@ type Span {
   spanKind: SpanKind!
   context: SpanContext!
 
-  """Span attributes as a JSON string"""
+  """Span attributes (excluding token count) as a JSON string"""
   attributes: String!
+
+  """Token count (total, prompt and completion)"""
+  tokenCount: TokenCount
```

Comment:
nesting it makes it a tad more complex of a structure 🤔 Any hesitation on just leaving it flat for simplicity, one less type?